src/ui/notification.js

```diff
@@ -2,17 +2,17 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui';
 
-const Notification = props => (
+const Notification = ({isDanger, ...rest}) => (
```

Comment:
Ah cool. I didn't know one can already "take apart" an object in the argument list. That's handy.

Reply:
Yeah, that's really nice. Learned this from Dr Axel:

https://exploringjs.com/impatient-js/ch_destructuring.html#where-can-we-destructure